src/Examine/LuceneEngine/Providers/LuceneIndex.cs

```diff
@@ -978,7 +1035,19 @@ private int ForceProcessQueueItems(bool block)
                 if (!RunAsync || block)
                 {
                     //commit the changes (this will process the deletes too)
-                    writer.Commit();
+                    if(_directory is ExamineDirectory examineDirectory)
+                    {
+                        //Calling commit causes fdt,fdx,fnm,frq,nrm,prx,tii,tis,tvd,tvf,tvx files to be deleted.
+                        //TODO: 
```

Comment:
Is this code comment still relevant? Commit shouldn't remove files IIRC, that is done when merging occurs